### PR TITLE
Fix C.I. scripts links

### PR DESCRIPTION
**Pull request by davcri**

### CODE REVIEW

File: build-emscripten.md [hunk 1]
```diff
@@ -61,6 +61,6 @@ If you are not using the `Ninja` generator then you can, since CMake 3.12, pass
 ### C.I. Scripts
 
 For reference you can have a look at the continuous integration scripts:
-- [nCine-libraries](https://github.com/nCine/nCine-libraries/blob/master/azure-pipelines.yml) (`Non_Windows` job)
-- [nCine](https://github.com/nCine/nCine/blob/master/azure-pipelines.yml) (`Emscripten` job)
-- [ncPong](https://github.com/nCine/ncPong/blob/master/azure-pipelines.yml) (`Emscripten` job)
+- [nCine-libraries](https://github.com/nCine/nCine-libraries/blob/master/.github/workflows/emscripten.yml)
+- [nCine](https://github.com/nCine/nCine/blob/master/.github/workflows/emscripten.yml) (`Emscripten` job)
```
Comment (encelo) [inline build-emscripten.md]: I don't think that a note about "`Emscripten` job" is useful anymore, as each workflow script now only covers a single platform.

File: build-emscripten.md [hunk 2]
```diff
@@ -61,6 +61,6 @@ If you are not using the `Ninja` generator then you can, since CMake 3.12, pass
...
+- [ncPong](https://github.com/nCine/ncPong/blob/master/.github/workflows/emscripten.yml) (`Emscripten` job)
```
Comment (encelo) [inline build-emscripten.md]: Same as the previous comment, "(`Emscripten` job)" note can be dropped.

**Review by encelo — CHANGES_REQUESTED** [review]
I would change the commit message to read "C.I." (with dots) as it has always been spelled this way in every previous message.

**Comment by davcri** [comment]
@encelo commit amended, now it should be fine!

**Review by encelo — APPROVED** [review]

**Pull request merged**